summernote/summernote
# v0.8.11

Improvement and bug fix
 - Implemented commit command (resets history but keeps editor content).
 - Add color picker support
 - Add start parameter for YouTube embedding
 - Add OnImageLinkInsert callback for handling inserted urls
 - Make the "range" object available for external modules.
 - When creating a link Summernote now checks maxTextLength
 - Add uglify for summernote-lite
 - Do not add default scheme for relative URLs
 - Update custom color selection logic
 - Move test file into the right location
 - Clean up test codes
 - Add styles for custom color selection
 - Add forecolor and backcolor buttons
 - Apply latest style changes into another
 - Do not wrap bs4 toolbar to prevent text flooding over toolbar UI
 - Do not wrap bs3 toolbar to prevent loosing the background shading
 - Add an example for placeholder text
 - Add an example for custom styletags
 - Drop debounce rate from 100ms to 10ms
 - Remove wrong executable attributes
 - Use default Bootstrap 4 checkbox style, not custom-control 
 - Make AutoLink module follows linkTargetBlank option
 - Use class instead of id for "open in new window" checkbox to avoid du… …
 - Add randomized id to link dialog checkbox
 - Move random id generation to Context
 - Revert the randomization changes
 - Prevent error by settings the default container
 - Less clutter by moving templates into .github
 - Allow html tags on placeholder
 - Allow protocol-less URL for YouTube video insertion
 - Make full screen mode properly even with maxHeight option
 - Add an example for Lite + Legacy Bootstrap
 - Add video and embed tag into the void element list
 - Fixed #1410 using a configurable blockquote breaking level.
 - FIX click on HintPopover suggestions doesnt insert
 - Fix issue #1964 with copy paste content in reverse order 
 - Fix wrong argument result due to the use of incorrect function type
 - Fix link autodetection  …
 - Fix dropdown toggle issue on lite while using with Bootstrap
 - Fix wrong lite style
 - Fix wrong bs4 checkbox
 - Fix expectContents to work with given context
 - Fix tests - to expect valid markup.
 - Fixed indentation of list items  …
 - Fixed outdent of nested list items.  …
 - Fix conflict by merging #1574
 - Fix indentation on checkbox creations
 - Fix Bootstrap4 background transparency issue #3024
 - Fix blocking form submitting issue #530
 - Fix minor things
 - Fix order of Airmode elements
 - Fix broken examples and update to use latest libraries
 - Fix class name for AirMode in lite.js

Translation
 - Improved translation in pt-PT for image, table, style and help sections
 - Improved translation in pt-BR for image section
 - Added missing new line at the end of file lang/summernote-pt-PT.js
 - pl-PL i18n update
 - Update summernote-nl-NL.js
 - Update summernote-pt-BR.js
 - Add missing translations in zh-CN for table, etc
 - Update summernote-de-DE.js  …
 - Fix pt-BR and gl-ES lang keys for style.p translations
 - Update summernote-th-TH.js
 - Missing translations with tabel (#3031)  …
 - Update Finnish translation
